src/backend/printers.service.ts

```diff
@@ -22,6 +23,47 @@ export class PrintersService extends BaseService {
     formData.apiKey = patch.apiKey;
   }
 
+  static applyPastedPrinterUrl(printerURL: URL, formData: PreCreatePrinter) {
+    formData.printerHostPort = parseInt(printerURL.port) || 80;
+    formData.printerHostName = printerURL.hostname;
+    formData.printerHostPrefix = printerURL.protocol.replace(":", "") as HttpProtocol;
+  }
+
+  static validPort(port: string | undefined) {
+    const isNumber = /^\d+$/;
+
+    if (port == undefined) return false;
+    try {
+      console.log(port);
```

Comment:
Please remove log entries like these

Reply:
Sorry i put them for testing and forgot to remove them